src/relay/transforms/dynamic_to_static.cc

```diff
@@ -55,6 +58,20 @@ class DynamicToStaticMutator : public MixedModeMutator {
         static const Op& op = Op::Get("tile");
         return Call(op, {call_node->args[0]}, Attrs(attrs), {});
       }
+    } else if (call_node->op == dyn_topk_op_) {
+      if (const ConstantNode* k = call_node->args[1].as<ConstantNode>()) {
```

Comment:
Maybe we should have `MakeTopK`, `MakeTile`, etc?

Reply:
It's an idea, those functions are available, we'd just have to pull them into headers.

Reply:
There are a lot of duplicated or referenced MakeOp functions in here: https://github.com/apache/incubator-tvm/blob/master/src/relay/transforms/pattern_util.h, I'll make a separate PR to pull those out into an op-level utility header and clean things up a bit.